Kitodo-LongTimePreservationValidationModule/src/main/java/org/kitodo/longtimepreservationvalidationmodule/ValidationResultState.java

```diff
@@ -13,52 +13,113 @@
 
 import edu.harvard.hul.ois.jhove.RepInfo;
 
+import java.util.Optional;
+
 import org.kitodo.api.validation.State;
+import org.kitodo.config.KitodoConfig;
+
+/**
+ * The validation result state is used to map the two-valued result state of
+ * JHove to the Kitodo validation interface.
+ */
+public class ValidationResultState {
```

Comment:
Is there some reason to change from enum to class with static final strings? It looks that better here would be change enum attribut to string but keep it as enum.

Reply:
I am not sure if I understood you correctly (linguistically). The previous enumeration type was hard-coded, simply because I did not have any input from the Product Owner on how to do it. Now there is, as ordered, a class for it to look up the mapping in the configuration. How could I have done otherwise?